### DISCUSSION TITLE
Instagram-like stories

### DISCUSSION THREAD

**Discussion by chetanyakan** [mattermost/community#93] [Enhancements]
Users could add their daily standup as a story-like video.
Stories should automatically disappear after 24 hours.

Similar to https://www.theverge.com/2020/10/7/21505519/slack-instagram-stories-push-to-talk-audio-calls-new-features#:~:text=Slack%20is%20getting%20ready%20to,before%20the%20end%20of%202020.&text=Slack%20sees%20the%20video%20snippets,daily%20agenda%20for%20a%20team.

**Comment by esethna** [comment]
That would be great for team meeting channels to keep things more async

**Reply by esethna** [reply]
I would almost categorize this as "practical" rather than entertainment ;)
